### DISCUSSION TITLE
NAnt usage in build process

### DISCUSSION THREAD

**Discussion by savornicesei** [nhibernate/nhibernate-core#3244] [General] [answered]
Hi all,

Is NAnt still used in building nhibernate? I can see the build script in the root of the repository but the build script(s) use psake.

Thanks,
Simo

**Comment by fredericDelaporte** [comment, marked as answer]
It has various usages within the [NHibernate build menu](https://github.com/nhibernate/nhibernate-core/blob/master/CONTRIBUTING.md#the-build-menu), like regenerating async code, launching tests, locally building the release package, ...

It is used for building locally the [documentation](https://github.com/nhibernate/nhibernate-core/blob/master/doc/documentation.build) too.

**Reply by savornicesei** [reply]
Great news, thanks!
Then I can use NHibernate project to smoke-test latest NAnt changes :partying_face:
